src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -185,14 +185,16 @@ def load_command_table(self, _):
 
     monitor_custom = CliCommandType(
         operations_tmpl='azure.cli.command_modules.monitor.custom#{}',
-        exception_handler=monitor_exception_handler)
+        exception_handler=monitor_exception_handler
+    )
```

Comment:
Which one is the recommended style? I am not sure. Can you give a reference?

Reply:
There's no recommendation. I think the style of the file should be consistent.